<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QDialog" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send</string>
  </property>
  <widget class="QLabel" name="sStatusLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>221</width>
     <height>17</height>
    </size>
   </property>
   <property name="text">
    <string>Please choose the file to be sent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sCloseBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>27</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="QPushButton" name="sSendBtn">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="sChooseLabel">
      <property name="text">
       <string>choose file/folder</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="sOpenBtn">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>29</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>open...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
